README.md

```diff
@@ -374,13 +376,43 @@ let app =
         ]
 ```
 
+### subRoute
+
+`subRoute` checks the current path begins with the `path` string, all `route*` and `subRoute*` handlers within the `handler` parameter will now be rooted to, and relative with respect to this path.
+
```

Comment:
Suggestion:

`subRoute` checks if the current path begins with the given `subPath` and will partially apply the route if it was a match. The passed in handler (and any nested handlers within it) should omit the already applied `subPath` for subsequent route evaluations.